src/config/index.ts

```diff
@@ -10,7 +10,7 @@ import {DevSecurityLevel} from '@shardus/core'
 
 crypto.init('69fa4195670576c0160d660c3be36556ff8d504725be8a59b5a96509e0c994bc')
 
-export const networkAccount = '0'.repeat(64)
+export const networkAccount = '1000000000000000000000000000000000000000000000000000000000000001'
```

Comment:
I'm assuming the 0.repeat(64) will be staking address like shardeum was?

Reply:
I was just setting it the new address because Shardeum use that and also the address is hardcoded in the archiver server. 
https://github.com/shardus/archive-server/blob/2fc70b7c4de0d6afa128ecffe20e5af29dede60b/src/Config.ts#L146